samples/csharp_dotnetcore/02.echo-bot/Startup.cs

```diff
@@ -48,7 +48,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseDefaultFiles();
             app.UseStaticFiles();
-
+            app.UseWebSockets();
```

Comment:
we do not want web sockets enabled by default.

Reply:
Not enabling WebSockets will mean the bot would stop working for DirectLine Speech. If a bot is unable to support both channels we have a big problem.